### DISCUSSION TITLE
Standalone executable or GUI installer for `frictionless-py`

### DISCUSSION THREAD

**Discussion by fjuniorr** [frictionlessdata/frictionlessdata.io#732] [Ideas]
Has anyone ever wanted either a standalone executable or a GUI installer (especially for windows) for `frictionless-py`? We are spreading knowledge around frictionless tooling in my organization for individuals responsible for datasets that don't have deep technical expertise and the need to install python and manage virtual environments is a big pain point for those that don't already do this. 

I can also think of technical users, such as people using R with Rstudio, that are comfortable enough to use `frictionless-py` in the CLI - and could benefit a lot from it - but don't want to deal with the hassle of managing a python installation. 

I'm also not sure that any web service (such as [Frictionless Repository](https://github.com/frictionlessdata/repository)) totally replaces local installations.

I've only found one issue [#497](https://github.com/frictionlessdata/frictionless-py/issues/497) in the community somewhat related to this but I'm very curious to hear more.

**Comment by roll** [comment]
Hi @fjuniorr,

Frictionless Application is on our roadmap - https://software.frictionlessdata.io/pages/roadmap.html (which will be an "installable" app). It will include work of creating `fricitonless-py` "binary". It even also can provide a CLI mode within application if there is a need for it
